spec/compiler/crystal/tools/doc/doc_renderer_spec.cr

```diff
@@ -275,5 +275,14 @@ describe Doc::Markdown::DocRenderer do
           %(<a href="Base.html#foo2(a,b)-instance-method">Base#foo2(a, a)</a> and <a href="Base.html#foo3(a,b,c)-instance-method">#foo3</a> and <a href="Base.html">Base</a>))
       end
     end
+
+    it "does not break when referencing lib type (#9928)" do
+      program = semantic("lib LibFoo; BAR = 0; end", wants_doc: true).program
+      generator = Doc::Generator.new(program, [""])
+
+      # TODO: There should not be a link to LibFoo::Bar in the first place
+      # because LibFoo is undocumented
+      assert_code_link(generator.type(program), "LibFoo::BAR", %(<a href="LibFoo.html#BAR">LibFoo::BAR</a>))
+    end
```

Comment:
@straight-shoota Looks like a botched merge.

Reply:
Oh, that's messed up

Reply:
Let's throw all that out with a `git rebase -i`, leaving only the original commit.

Reply:
Sorry for the mess. I made a hard reset and a clean merge of master.